OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs

```diff
@@ -167,7 +167,7 @@ public override void DrawOuter()
 			Game.Renderer.EnableScissor(drawBounds);
 
 			drawBounds.Offset((-ChildOrigin).ToPoint());
-			foreach (var child in Children)
+			foreach (var child in Children.ToArray())
```

Comment:
Hmm, I tried adding `Game.RunAfterTick(() => { });` everywhere, but that kinda had an opposite effect.